x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/create/index.tsx

```diff
@@ -273,151 +285,155 @@ const CreateRulePageComponent: React.FC = () => {
 
   return (
     <>
-      <WrapperPage restrictWidth>
```

Comment:
NOTE: I did _not_ change a bunch of code here.

Instead I just changed:

```ts
<WrapperPage restrictWidth>
       ....
```

to be:

```ts
<WrapperPage>
        <EuiFlexGroup direction="row" justifyContent="spaceAround">
          <MaxWidthEuiFlexItem>
                ....
```